lib/updateThemeCss/saveFile.ts

```diff
@@ -0,0 +1,38 @@
+
+import SparkMD5 from 'spark-md5'
+import prisma from '../prisma';
+const Minio = require('minio')
+import { Readable } from 'stream'
+
+
+export default async function saveFile(cssContent, subdomain, pubkey) {
+  const minioClient = new Minio.Client({
+      endPoint: 's3.amazonaws.com',
+      accessKey: process.env.AWS_ACCESS_KEY,
+      secretKey: process.env.AWS_SECRET_KEY
+  });
+
+
+
+  const hash = SparkMD5.hash(cssContent)
+  const location = `${subdomain}.holaplex.com/${pubkey}/overrides-${hash}.css`
```

Comment:
```suggestion
  const location = `${pubkey}/overrides-${hash}.css`
```
The pubkey is all we need.